### DISCUSSION TITLE
Walsafe 基于walrus的加密存储平台

### DISCUSSION THREAD

**Discussion by 0xFANGO** [creatorsand/.github#122] [Ideas]
**一句话 IDEA：**

WalSafe是一个基于 Walrus 服务的，去中心化的加密存储平台

**代码仓库**
https://github.com/SilenceRichard/wal-safe/tree/main

**DEMO链接**
https://wal-safe.vercel.app/

**成员角色：**

前端：方格

合约开发: 野兔

**成员列表**

1.  Funky Ge
2. 歪脖山的野兔

**个人简介：**

由一位经验丰富的互联网资深前端和一位技术全面的合约开发者组成的团队，兼具强大的前端开发能力与深厚的区块链技术背景，能够高效构建优质的去中心化应用

**产品是给谁用的？**

适合想加密存储私人数据的用户，确保这些信息不被未经授权的访问者获取，避免被中心化平台泄露或删除。

**产品亮点**
    
- **加密文件保护隐私**
    
    为用户提供基于文件加密功能，数据仅对授权用户可见，充分保障隐私安全，避免敏感信息泄露。

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]
